go.mod

```diff
@@ -5,13 +5,13 @@ go 1.19
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.6
 	github.com/Masterminds/semver/v3 v3.2.0
-	github.com/Microsoft/go-winio v0.6.0
+	github.com/Microsoft/go-winio v0.6.1
 	github.com/ProtonMail/go-crypto v0.0.0-20230217124315-7d5c6f04bbb8
 	github.com/YourFin/binappend v0.0.0-20181105185800-0add4bf0b9ad
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
 	github.com/cavaliergopher/grab/v3 v3.0.1
 	github.com/cheggaaa/pb/v3 v3.1.0
-	github.com/containers/gvisor-tap-vsock v0.5.0
+	github.com/containers/gvisor-tap-vsock v0.6.2-0.20230609094134-b79db04d368e
```

Comment:
I'd prefer to stick to release gvisor-tap-vsock versions.

Reply:
@cfergeau once we have the next release cut we will use it but as of now we have a old release.

Reply:
this fixes a critical bug on CRC, it'd be good to vendor a released gvisor-tap-vscok but since we don't have a newer release with the needed changes, we can do that when there's new release for gvisor-tap-vsock